torch/csrc/jit/runtime/static/generated_ops.cpp

```diff
@@ -783,12 +783,12 @@ REGISTER_OPERATOR_FUNCTOR(aten::dot, aten_dot, [](Node* n) -> SROperator {
       const auto& self = p_node->Input(0).toTensor();
       const auto& tensor = p_node->Input(1).toTensor();
       if (p_node->Output(0).isNone()) {
-        p_node->Output(0) = at::native::dot(self, tensor);
+        p_node->Output(0) = at::cpu::dot(self, tensor);
```

Comment:
I'm curious, how did the namespace change?

Reply:
That's just the namespace that the structured ops generate the kernels into. Compare with erf below which is also structured. The `at::native::dot` function is no longer required as the device guard and other functionality has moved to `META_FUNC`